cmd/keytransparency-sequencer/main.go

```diff
@@ -147,11 +137,11 @@ func main() {
 	defer done()
 
 	spb.RegisterKeyTransparencySequencerServer(grpcServer, sequencer.NewServer(
-		directoryStorage,
+		db.Directories,
 		trillian.NewTrillianLogClient(lconn),
 		trillian.NewTrillianMapClient(mconn),
 		trillian.NewTrillianMapWriteClient(mconn),
-		mutations, mutations,
+		db.Batches, db.Logs,
```

Comment:
nit: I feel like separate lines for these 2 params would be more consistent. Unless I'm missing something implicit about why they're on the same line?

Reply:
You're right.